main/src/ca/uwaterloo/flix/language/dbg/DocAst.scala

```diff
@@ -327,6 +327,10 @@ object DocAst {
 
     case class Native(clazz: Class[_]) extends Atom
 
+    case class JvmConstructor(method: Method) extends Atom
+
+    case class JvmMethod(constructor: Constructor[_]) extends Atom
```

Comment:
You swapped them :)

Constructor should be constructor.
Method should be method.